AdvPromptSet/utils.py

```diff
@@ -0,0 +1,187 @@
+import os
+import numpy as np
+import pandas as pd
+
+
+def process_jigsaw_1(dat_folder):
```

Comment:
could be good for future reference to add docstrings saying quickly what each of these do in a nutshell

Reply:
Make sense! added docstrings for all functions